corehq/blobs/metadata.py

```diff
@@ -70,7 +70,7 @@ def delete(self, key, content_length):
         :returns: The number of metadata rows deleted.
         """
         with BlobMeta.get_plproxy_cursor() as cursor:
-            cursor.execute('SELECT 1 FROM delete_blob_meta(%s)', [key])
+            cursor.execute('SELECT key FROM blobs_deletedblobmeta where key = %s', [key])
```

Comment:
I think this query has a different outcome than the previous query. The previous query ran a function that deleted blob metadata (and saved copies into `blobs_deletedblobmeta`). The new query simply runs a select that does not change database state.

Reply:
Noted with thank will take a look at my environment once again.